spec/features/articles_list_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+describe "Listing articles" do
+  scenario "someone views the articles index" do
+    visit articles_path
+
+    expect(page).to have_content "Articles List"
```

Comment:
you should add a second test that creates an article and verifies that its title is in the list.

You can use the FFaker gem to generate 3 paragraphs of text.
You should run the test with a save_and_open_page, atleast once, to ensure expected results